### DISCUSSION TITLE
Release schedule

### DISCUSSION THREAD

**Discussion by nicolinux** [outline/outline#5856] [Self Hosting] [answered]
Hi, I see that the tags 0.72.0 are starting to pop up and I’d like to ask if a new release is imminent. Reason I am asking is because I would like to deploy Outline Wiki (fresh install) and would like to avoid updating a few days later. 
Thanks for the awesome project btw. The wiki is a joy to use and coming from Confluence with its awesome editor, it is nice to see how Outline Wiki matches and even surpasses it!

**Comment by tommoor** [comment, marked as answer]
Probably next week, but updating is easy.

If you want an easier time installing and updating you can run the pre-release ([extra instructions](https://github.com/outline/outline/issues/5716#issuecomment-1722312302)).
